ghost/announcement-bar-settings/package.json

```diff
@@ -21,6 +21,5 @@
     "c8": "8.0.1",
     "mocha": "10.2.0",
     "sinon": "15.2.0"
-  },
-  "dependencies": {}
+  }
```

Comment:
This gave me an idea 😄 thank you!

https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/issues/683